interface/main/tabs/js/tabs_view_model.js

```diff
@@ -294,7 +293,7 @@ function menuActionClick(data,evt)
             var encounterID=app_view_model.application_data[attendant_type]().selectedEncounterID();
             if(isEncounterLocked(encounterID))
             {
-                alert(xl_strings_tabs_view_model.encounter_locked);
+                alert(xl('This encounter is locked. No new forms can be added.'));
```

Comment:
It appears that spacing or period is breaking the translations. Looking into this. TODO